src/elements/Button/Button.js

```diff
@@ -250,8 +251,11 @@ class Button extends Component {
     }
 
     if (label) {
-      const classes = cx('ui', baseClasses, 'button', className)
-      const containerClasses = cx('ui', labeledClasses, 'button', className)
+      let classes = cx('ui', baseClasses, 'button', className)
+      if (className && className.indexOf('disabled') !== -1) {
+        classes = cxd(classes, { disabled: false })
```

Comment:
This line here is failing Codecov as it is not invoked during tests.  That said, we don't support component features through className but only props.  I believe this should be removed.  Let me know if you believe otherwise and why.

>Here is the coverage hint from the Codecov browser extension:
>
>![image](https://cloud.githubusercontent.com/assets/5067638/21526540/db199726-ccd9-11e6-883c-70a0f9474b35.png)